.github/workflows/python-package.yml

```diff
@@ -24,40 +24,27 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
+    - name: Install pip
+      run: python -m pip install --upgrade pip
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip
-        python -m pip install flake8
+        python -m pip install --upgrade setuptools wheel twine
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
         if [ -f requirements-dev.txt ]; then pip install -r requirements-dev.txt; fi
-    - name: Lint with flake8
-      run: |
-        # TODO: stop the build if there are Python syntax errors or undefined names
-        flake8 . --count --exit-zero --select=E9,F63,F7,F82 --show-source --statistics
-        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
-    - name: Install package
-      run: |
-        python setup.py install
-    - name: Download certificate and private key
```

Comment:
A  certificate is required and many other config files. Please ensure the initial github action script works as expected. And please before removing any code, you can always enquire why it's there so you can make necessary changes if they need be